docker-compose.yaml

```diff
@@ -4,7 +4,7 @@
 #
 services:
   keycloak:
-    image: quay.io/keycloak/keycloak:25.0.6@sha256:82c5b7a110456dbd42b86ea572e728878549954cc8bd03cd65410d75328095d2
+    image: quay.io/keycloak/keycloak:24.0.5@sha256:c916c668a5cd589948c3310ab31ffcfc5da55f0e546028f2f606419ce17c6ad8
```

Comment:
OMG. rollback.. ;-)

Probably good to stick with our focus on PodiumD 2.0 for now but for the future I do think we need to rethink this because this way we will create more big-bang upgrades when switching our focus from PodiumD version X to Y. And that is exactly what we wanted to avoid by these automated upgrades.. 

And of course there is the angle: for open source development you want to be on the latest versions in general.

Reply:
Actually I'm not sure we need to or even should rollback this Keycloak upgrade for our Docker Compose setup. Because this newer version of Keycloak runs just fine on our ZAC code and test environment and this is because Keycloak is usually very careful not to introduce backwards compatibility issues (but you could doublecheck in the release notes of Keycloak). I would personally leave it and pin it at this 25 version, at least for our Docker Compose update. 

I think what we needed to do to upgrade back then was only to change that Keycloak health endpoint in our integration test code or something. But you could check I guess.

Reply:
There will be a PodiumD 3 release available soon, and after that a 4 release with ZAC in it. We will push for a new (latest maybe) version of Keycloak for the 4 release, so let's just go forward with this for now. We'll come back to this in a month or so.